source/grid/tria.cc

```diff
@@ -1195,7 +1195,8 @@ namespace internal
       types::boundary_id,
       << "The input data for creating a triangulation contained "
       << "information about a line with indices " << arg1 << " and " << arg2
-      << " that is described to have boundary indicator " << (int)arg3
+      << " that is described to have boundary indicator "
+      << static_cast<int>(arg3)
```

Comment:
not needed any more since `boundary_id` is now an integer type and no longer a `char` type

Reply:
Hm, I see. Sure, I don't feel strongly either way.